mobile-menu.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-multi-str */
+/* eslint-disable no-plusplus */
+const hdr = document.querySelector('.pg-hdr'); // the header element
+const btn = document.querySelector('.btn'); // the opening button
+const logo = document.querySelector('.pg-hdr h3'); // The logo name
+const navBar = document.querySelector('.dsk-btn'); // The desktop navigation bar
+const navLinks = document.querySelectorAll('.nav-link'); // The navigation links inside the nav bar
+const rightArrow = document.querySelectorAll('.r-arrow'); // The right arrow after the nav links
+const navIcon = document.querySelector('.nav-icon'); // The mail icon
+
+/** ******* The closig button *********** */
+const cross = document.createElement('button'); // The cross button to close mobile menu
+cross.classList.add('closing-button');
+cross.innerHTML = '&times;';
+
+function removed(e) {
```

Comment:
- [ ] OPTIONAL
It will be a nice idea to prefer the use of `arrow function` over normal function, so as to keep your function syntax simpler and cleaner.

Reply:
Thanks, I'll keep it in mind!